src/server.c

```diff
@@ -3143,7 +3164,7 @@ void call(client *c, int flags) {
 
     /* Call the command. */
     dirty = server.dirty;
-    prev_err_count = server.stat_total_error_replies;
+    incrCommandFailedCalls(NULL, 0);
```

Comment:
i'm not certain this is needed, but i suppose it can't hurt.
i.e. otherwise, there's maybe a chance a previous error will be counted below.
but on the other hand, can't this cause some error to not get counted at all?

Reply:
This is how it was before: `prev_err_count = server.stat_total_error_replies`, do you want to change it?